src/App.js

```diff
@@ -5,15 +5,23 @@ import { COUNTER, setLocalData, getLocalData } from "./utils/storage.js";
 
 const App = () => {
 	const [count, setCount] = useState(0);
-
 	const [counterText, setCounterText] = useState("");
+	const [isDisabled, setDisabled] = useState(false);
 
 	const handleCountAddition = () => {
 		setCount(prevCount => prevCount + 1);
 	};
 
 	const handleCountSubtract = () => {
-		setCount(prevCount => prevCount - 1);
+		if (count === 1) {
+			alert("hi");
```

Comment:
Please remove unwated things/debuggin code and unwanted comments and clean the file